bin/summariser.py

```diff
@@ -101,6 +102,20 @@ def runprocess(db_config_file, config_file, log_config_file):
         sys.exit(1)
 
     log.info('Created Pidfile')
+
+    # Configure options for stale summary clean up.
+    try:
+        stale_summary_clean_up = cp.getboolean('summariser',
+                                               'delete_stale_summaries')
+
+        stale_summary_window_days = cp.getint('summariser',
+                                              'stale_summary_window_days')
```

Comment:
How come stale_summary_window_days passes itself?

Reply:
The string `'stale_summary_window_days'` is refers to the option in the config file that `cp.getint(...` is reading from. Though yes, I will change the varibale name to... something else.

Reply:
addressed :)

Reply:
in https://github.com/apel/apel/pull/222/commits/e3c46d72e3e710b1fe2b3cab31187802c58a9c2f